main.py

```diff
@@ -413,9 +413,10 @@ def get_redirect_route(
         r'/<story_url_fragment>' % feconf.REVIEW_TEST_DATA_URL_PREFIX,
         review_tests.ReviewTestsPageDataHandler),
     get_redirect_route(
-        r'%s/review-test/<story_url_fragment>'
-        % feconf.TOPIC_VIEWER_URL_PREFIX,
-        review_tests.ReviewTestsPage),
+         r'%s/can_access_review_tests_page/<classroom_url_fragment>'
+         r'/<topic_url_fragment>/<story_url_fragment>' 
```

Comment:
Ditto instead of this nested url, maybe have a query string? (as this is going to be an API request and not a browser URL)

Reply:
I have tried to build on the other ACCESS_VALIDATION redirections and it was all using the same format of URLs. I dnt see why we need to change it here. @DubeySandeep